content/en/docs/concepts/containers/images.md

```diff
@@ -150,8 +163,8 @@ example, run these on your desktop/laptop:
    1. Get a list of your nodes, for example:
       - if you want the names: `nodes=$(kubectl get nodes -o jsonpath='{range.items[*].metadata}{.name} {end}')`
       - if you want to get the IPs: `nodes=$(kubectl get nodes -o jsonpath='{range .items[*].status.addresses[?(@.type=="ExternalIP")]}{.address} {end}')`
-   1. Copy your local `.docker/config.json` to the home directory of root on each node.
-      - for example: `for n in $nodes; do scp ~/.docker/config.json root@$n:/root/.docker/config.json; done`
+   1. Copy your local `.docker/config.json` to one of the search paths list above.
+      - for example: `for n in $nodes; do scp ~/.docker/config.json root@$n:/var/lib/kubelet/config.json; done`
```

Comment:
maybe make the example a path that is always consulted, regardless of envvars, like `/.docker/config.json`?

Reply:
👍 to @liggitt

Reply:
I am not sure `/.docker/config.json` is the perfect place, in some situations, the `/` is read-only.. I can see two reasons for writing to the root dir of kubelet:

1. make the scope of docker config as minimal.
2. make it clear that this config.json is supposed to be consumed by kubelet.